### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AzureLeon1**
1. Add 4 new papers, and update the tag `number of papers` from 211 to 215. The newly added papers are:
    - A Survey on Masked Autoencoder for Visual Self-supervised Learning, IJCAI 2023
    - Masked Autoencoders for Point Cloud Self-Supervised Learning, ECCV 2022
    - Masked Autoencoder for Self-Supervised Pre-Training on Lidar Point Clouds, WACVW 2023
    - Joint-MAE: 2D-3D Joint Masked Autoencoders for 3D Point Cloud Pre-training, IJCAI, 2023
2. Remove 3 duplicate entries, leaving only one entry for each work:
    - Masked Discrimination for Self-Supervised Learning on Point Clouds
    - GD-MAE: Generative Decoder for MAE Pre-training on LiDAR Point Clouds
    - Learning 3D Representations from 2D Pre-trained Models via Image-to-Point Masked Autoencoders
3. Put the papers related to the point cloud into a separate group, considering that there are enough related papers.

### CODE REVIEW

**Comment by EdisonLeeeee** [comment]
Thank you!
